prune_shard_list.c

```diff
@@ -305,7 +304,7 @@ LookupOperatorByType(Oid typeId, Oid accessMethodId, int16 strategyNumber)
  * GetOperatorByType returns the operator oid for the given type, access
  * method, and strategy number.
  */
-static Oid
+Oid
```

Comment:
I think we're starting to see we might want to move some of these common functions out into their own file.